gooseai/generation/generation.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
 // 	protoc-gen-go v1.26.0
-// 	protoc        v3.21.4
+// 	protoc        v3.12.2
```

Comment:
Seems like this is using the older protoc, would like to stay with the more recent version.